src/test/java/spoon/test/template/testclasses/EnumAccessTemplate.java

```diff
@@ -13,15 +12,15 @@ public void method() throws Throwable {
 	}
 	
 	@Parameter("_AnEnum_")
-	CtTypeReference<?> anEnum;
+	String __AnEnum_;

```

Comment:
Super interesting, you're right. Let's relax the contract and remove the naming convention.